packages/pn-pa-webapp/src/redux/auth/__test__/test-utils.ts

```diff
@@ -28,8 +28,8 @@ export const userResponse: User = {
   name: 'Mario',
   family_name: 'Rossi',
   fiscal_number: 'RSSMRA80A01H501U',
-  email: 'mocked-email',
-  uid: 'mocked-uid',
+  email: 'mocked-email@mail.com',
```

Comment:
Please add some comment about the convenience of defining these mock data according to the format specifications.

Reply:
Done